simbad/util/matthews_coef.py

```diff
@@ -1,71 +1,40 @@
 """Module for running matthews coefficients to calculate solvent content"""
 
 __author__ = "Adam Simpkin"
-__date__ = "07 Oct 2017"
-__version__ = "0.1"
+__date__ = "10 Oct 2017"
+__version__ = "0.2"
 
 from cctbx.crystal import symmetry
 from mmtbx.scaling.matthews import density_calculator
 from mmtbx.scaling.matthews import matthews_rupp
-from simbad.util import molecular_weight
+from simbad.util.pdb_util import PdbStructure
 
 
 class SolventContent(object):
     def __init__(self, cell, sg):
         self.crystal_symmetry = symmetry(unit_cell=cell, space_group_symbol=sg)
         self.dens_calc = density_calculator(self.crystal_symmetry)
 
-    def calculate(self, pdb):
-        return self.dens_calc.solvent_fraction(molecular_weight(pdb), 0.74) * 100
+    def calculate_form_file(self, pdb):
```

Comment:
This may be fixed in a latter commit, but should be calculate_from_file

Reply:
Good spot!

Reply:
Sorted in 5e1c71d